File: JavaScript_Courses/JavaScript_Basics/Preparation/06_Nested_Loops_Lab/06_Building.js
function building(input) {
    // две цели числа - броят на етажите и броят на стаите за един етаж. 
    let floors = Number(input[0]);
    let rooms = Number(input[1]);

    for (let i = floors; i > 0; i--) {
        let result = '';
        for (let j = 0; j < rooms; j++) {
            if (i == floors) {
                result += `L${i}${j} `;
            } else if (i % 2 == 0) {
                result += `O${i}${j} `;
            } else if (i % 2 == 1) {
                result += `A${i}${j} `;
            }
        }

        console.log(result);
    }
}


// building(["6", "4"]);

File: Python_Courses/Python_Basics/Preparation/06_Nested_Loops_Lab/06_Building.py
floors = int(input())
rooms = int(input())

for i in range(floors, 0, -1):
    for j in range(rooms):
        if i == floors:
            print(f"L{i}{j} ", end="")
        elif i % 2 == 0:
            print(f"O{i}{j} ", end="")
        elif not i % 2 == 0:  # else:
            print(f"A{i}{j} ", end="")
    print()
